app/javascript/crayons/Icons/Icon.jsx

```diff
@@ -0,0 +1,25 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+
+export const Icon = (props) => {
+  const { src: InternalIcon, native, className, ...otherProps } = props;
+
+  return (
+    <InternalIcon
+      className={classNames('crayons-icon', {
+        'crayons-icon--default': native,
+        [className]: className,
+      })}
+      {...otherProps}
+    />
+  );
+};
+
+Icon.displayName = 'Icon';
+
+Icon.propTypes = {
+  native: PropTypes.bool,
+  className: PropTypes.string,
+  src: PropTypes.ReactNode,
```

Comment:
```suggestion
  src: PropTypes.ReactNode.isRequired,
```
I guess it won't work without a `src` so we can make it a required prop 🙂